tests/operator/operator.go

```diff
@@ -2070,7 +2055,7 @@ spec:
 			Expect(handlerImageName).To(ContainSubstring(flags.ImagePrefixAlt), "virt-handler should have correct image prefix")
 
 			By("Verifying VMs are working")
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+			vmi := libvmifact.NewCirros()
```

Comment:
I don't really care, but originally it was Alpine. Intentional?

Reply:
this is a mistake. but if it will pass the e2e, I'll leave it as it's now